Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM bitnami/postgresql:latest
+FROM bitnami/postgresql:10
```

Comment:
Good catch, `latest` definitely could have had unintended side-effects.

Reply:
Yeah, I changed it for my other needs and forgot to change it back. Nice catch!